configcatclient/configcatclient.py

```diff
@@ -318,8 +318,8 @@ def set_online(self):
         """
         if self._config_service:
             self._config_service.set_online()
-
-        self.log.debug('Switched to ONLINE mode.')
+        else:
+            self.log.warning('Client is configured to use the `%s` override behavior, thus `%s()` has no effect.', 'LocalOnly', 'set_online', event_id=3202)
```

Comment:
Why are we using %s here instead of putting the set_online directly into the string literal?